src/components/SettingsDialog/SettingsDialog.vue

```diff
@@ -152,7 +152,7 @@ export default {
 				.build()
 			picker.pick()
 				.then(async(path) => {
-					console.debug(`Path '${path}' selected for talk attachments`)
+					console.debug('Path '${path}' selected for talk attachments')
```

Comment:
This breaks the log that is generated as path is not evaluated anymore

Reply:
I corrected :wink:.